webview/src/experiments/components/table/styles.module.scss

```diff
@@ -590,70 +468,76 @@ th {
       background-color: $row-border-selected-color;
     }
   }
+}
 
-  &:hover:not(.rowSelected) {
-    td:not(.experimentCell),
-    .experimentCell::before {
-      background-color: $row-hover-background-color;
-    }
+.nestedRow {
+  .experimentCell .innerCell {
+    padding-left: calc($nested-row-padding + $edge-padding);
 
-    td:not(.experimentCell):hover::before,
-    td:hover + td::before {
-      background-color: $border-color;
+    .rowActions {
+      left: calc(($cell-padding + $nested-row-padding) * -1);
     }
   }
 }
 
-.workspaceRowGroup {
-  border: none;
-  position: sticky;
-  top: var(--table-head-height);
-  z-index: 4;
-  background-color: $row-bg-color;
-
-  &.withShadow {
-    box-shadow: 0 5px 8px -2px $shadow;
+.previousCommitsRow {
+  border-bottom: $row-border;
 
-    tr {
-      border-bottom: none;
-    }
+  td {
+    font-size: 0.6rem;
+    padding-left: 16px;
   }
+}
 
-  .innerCell {
-    padding: 5px 10px;
+.rowGroup:last-child {
+  & > tr:last-child {
+    border-color: $row-bg-color;
 
-    .rowActions {
-      visibility: hidden;
+    &.rowSelected {
+      border-color: $row-bg-selected-color;
     }
   }
 
-  .innerCell,
-  .timestampInnerCell {
-    height: 100%;
-    width: 100%;
-    background-color: transparent;
-  }
+  .experimentGroup {
+    &:last-child tr:last-child {
+      border-color: $row-bg-color;
 
-  .timestampInnerCell {
-    height: 42px;
+      &.rowSelected {
+        border-color: $row-bg-selected-color;
+      }
+    }
   }
 }
 
-.workspaceChange,
-.depChange {
-  color: $changed-color;
-}
-
 td {
   height: auto;
   font-size: 0.8rem;
   line-height: 2rem;
   align-items: center;
+  background-color: $row-bg-color;
 
   &:not(.experimentCell)::before {
     @extend %cellBorderLeft;
   }
 
+  .rowSelected & {
```

Comment:
I think it's a little better but the way we update backgrounds/borders for cells is still kind of complex. I think there are some better ways to go about this but I've already spent quite a bit of time on updating table styles, so will look at it at a later date!